app/controllers/memberships_controller.rb

```diff
@@ -0,0 +1,31 @@
+class MembershipsController < ApplicationController
+  before_action :user_belongs_to_group, only: [:destroy]
+  def new
+    @group = find_group
+    @group.add_user(current_user)
+
+    redirect_to @group
+  end
```

Comment:
@tabfugnic so with a many to many relationship set in place is this the best way to do this?